Dockerfile

```diff
@@ -1,30 +1,48 @@
-FROM ubuntu:bionic
+FROM runtimeverificationinc/ubuntu:bionic
 
-RUN    apt-get update         \
-    && apt-get upgrade --yes  \
-    && apt-get install --yes  \
-            bison             \
-            clang-8           \
-            cmake             \
-            flex              \
-            libboost-test-dev \
-            libcrypto++-dev   \
-            libgmp-dev        \
-            libjemalloc-dev   \
-            libmpfr-dev       \
-            libprocps-dev     \
-            libprotobuf-dev   \
-            libsecp256k1-dev  \
-            libssl-dev        \
-            libyaml-dev       \
-            pkg-config        \
-            protobuf-compiler \
-            rapidjson-dev     \
+RUN    apt-get update                \
+    && apt-get upgrade --yes         \
+    && apt-get install --yes         \
+            binutils-dev             \
+            bison                    \
+            clang-8                  \
+            cmake                    \
+            flex                     \
+            g++                      \
+            git                      \
+            gperf                    \
+            libboost-all-dev         \
+            libcap-dev               \
+            libcrypto++-dev          \
+            libdouble-conversion-dev \
+            libevent-dev             \
+            libgflags-dev            \
+            libgmp-dev               \
+            libgoogle-glog-dev       \
+            libiberty-dev            \
+            libjemalloc-dev          \
+            libkrb5-dev              \
+            liblz4-dev               \
+            liblzma-dev              \
+            libmpfr-dev              \
+            libnuma-dev              \
+            libprocps-dev            \
+            libprotobuf-dev          \
+            libsasl2-dev             \
+            libsecp256k1-dev         \
+            libsnappy-dev            \
+            libsodium-dev            \
+            libssl-dev               \
+            libtool                  \
+            libyaml-dev              \
+            libzstd-dev              \
+            make                     \
+            pkg-config               \
+            protobuf-compiler        \
+            rapidjson-dev            \
+            sudo                     \
+            unzip                    \
+            wget                     \
```

Comment:
The extra dependencies I see added here are:

```
binutils-dev
g++
git
gperf
libboost-all-dev
libcap-dev
libdouble-conversion-dev
libevent-dev
libgflags-dev
libgoogle-glog-dev
libiberty-dev
libkrb5-dev
liblz4-dev
liblzma-dev
libnuma-dev
libsasl2-dev
libsnappy-dev
libsodium-dev
libtool
libzstd-dev
make
sudo
unzip
wget 
```

Do (with `libboost-test-dev => libboost-all-dev`).

Do we know that they're all actually needed? For example, I see no use of `unzip` or `wget` in the repository.

Reply:
I think that they are required in the Jenkins stage that builds the `proxygen` dependency, but now I see that they are installed in the same script that is run in the stage: https://github.com/facebook/proxygen/blob/master/proxygen/build.sh#L28-L59